data-lineage/src/main/java/com/wipro/ats/bdre/lineage/LineageMain.java

```diff
@@ -434,17 +362,15 @@ private void fetchAllTokTableOrCol(ASTNode nodeSelExpr, String functionString, L
 
 			if (nodeSelExpr.getChild(j) != null && nodeSelExpr.getChild(j).getType() == HiveParser.TOK_TABLE_OR_COL) {
 				// if "." is present one step up then it is a table else a column
-				String tableName = null;
+
 				String colName = null;
 				if (nodeSelExpr.getChild(j).getParent().getType() == HiveParser.DOT) {
-					tableName = nodeSelExpr.getChild(j).getChild(0).toString();
 					colName = nodeSelExpr.getChild(j+1).toString();
 				} else {
 					colName = nodeSelExpr.getChild(j).getChild(0).toString();
 				}
-				String columnFullName = (tableName!=null ? tableName+"." : "") + colName;
 
-				System.out.println("Input column(infunction) = " + colName);
+				LOGGER.info("Input column(infunction) = " + colName);
 				columnList.addToList("inputcol(infunction)->" + colName);
 				inColumnList.add(colName);
 				functionString += ";inputcolumn->" + colName;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Introduce a new variable instead of reusing the parameter "functionString". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AS1226)